packages/fastify/src/withClerkMiddleware.ts

```diff
@@ -1,39 +1,40 @@
-import { parse } from 'cookie';
+import { createIsomorphicRequest } from '@clerk/backend';
 import type { FastifyReply, FastifyRequest } from 'fastify';
 
 import { clerkClient } from './clerkClient';
 import * as constants from './constants';
 import type { ClerkFastifyOptions } from './types';
 import { getSingleValueFromArrayHeader } from './utils';
```

Comment:
🔧 This also feels it belongs to the request utility

Reply:
What do you mean exactly? This is the `fastify` package importing the utility from backend as we have designed with @nikosdouvlis